easybuild/easyblocks/s/suitesparse.py

```diff
@@ -70,24 +72,32 @@ def configure_step(self):
             'LAPACK': os.getenv('LIBLAPACK_MT'),
         }
 
+        # Get METIS or ParMETIS settings
         metis = get_software_root('METIS')
         parmetis = get_software_root('ParMETIS')
         if parmetis:
             metis_path = parmetis
-            metis_libs = ' '.join([
-                os.path.join(parmetis, 'lib', 'libparmetis.a'),
-                os.path.join(parmetis, 'lib', 'metis.a'),
-            ])
+            metis_include = os.path.join( parmetis, 'include')
+            metis_libs = os.path.join(parmetis, get_software_libdir('ParMETIS'), 'libmetis.a')
```

Comment:
Yes, but don't you want to instruct `SuiteSparse` to link to `libparmetis.a`? (the link to `libmetis.a` is then also required to resolve all symbols, I think)